src/components/oculus-touch-controls.js

```diff
@@ -0,0 +1,259 @@
+var registerComponent = require('../core/component').registerComponent;
+var bind = require('../utils/bind');
+var trackedControlsUtils = require('../utils/tracked-controls');
+
+const TOUCH_CONTROLLER_MODEL_BASE_URL = 'https://cdn.aframe.io/controllers/oculus/';
+const TOUCH_CONTROLLER_MODEL_OBJ_URL_L = TOUCH_CONTROLLER_MODEL_BASE_URL + 'oculus_cv1_controller_left.obj';
+const TOUCH_CONTROLLER_MODEL_OBJ_MTL_L = TOUCH_CONTROLLER_MODEL_BASE_URL + 'oculus_cv1_controller_left.mtl';
+const TOUCH_CONTROLLER_MODEL_OBJ_URL_R = TOUCH_CONTROLLER_MODEL_BASE_URL + 'oculus_cv1_controller_right.obj';
+const TOUCH_CONTROLLER_MODEL_OBJ_MTL_R = TOUCH_CONTROLLER_MODEL_BASE_URL + 'oculus_cv1_controller_right.mtl';
+
+const GAMEPAD_ID_PREFIX = 'Oculus Touch';
+
+const FAKE_TOUCH_THRESHOLD = 0.00001;
+
+/**
+ * Oculus Touch Controls Component
+ * Interfaces with Oculus Touch controllers and maps Gamepad events to
+ * common controller buttons: trackpad, trigger, grip, menu and system
+ * It loads a controller model and highlights the pressed buttons
+ */
+module.exports.Component = registerComponent('oculus-touch-controls', {
+  schema: {
+    hand: {default: 'left'},
+    buttonColor: { default: '#FAFAFA' },  // Off-white.
+    buttonHighlightColor: {default: '#22D1EE'},  // Light blue.
+    model: {default: true},
+    rotationOffset: {default: 0} // no default offset; -999 is sentinel value to auto-determine based on hand
```

Comment:
Do you need this to be configurable from the outside? Can you just infer from the `hand` property internally?